<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serial_window</class>
 <widget class="QMainWindow" name="serial_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>809</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>809</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>809</width>
     <height>491</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>411</width>
        <height>281</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>接收窗口</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="3">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="minimumSize">
          <size>
           <width>391</width>
           <height>221</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="clear">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>清空接收</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>138</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="checkBox">
         <property name="maximumSize">
          <size>
           <width>77</width>
           <height>19</height>
          </size>
         </property>
         <property name="text">
          <string>HEX显示</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>466</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>串口设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>端口号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="PortBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>波特率：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="BaudBox">
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>57600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4800</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>数据位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="BitNumBox">
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>校验位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="ParityBox">
             <item>
              <property name="text">
               <string>无</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>奇校验</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>偶校验</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>停止位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="StopBox">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="minimumSize">
          <size>
           <width>164</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>169</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>157</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>164</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>169</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="minimumSize">
          <size>
           <width>164</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>169</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="openButton">
         <property name="minimumSize">
          <size>
           <width>209</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>411</width>
        <height>181</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>发送窗口</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="4">
        <widget class="QTextEdit" name="textEdit">
         <property name="minimumSize">
          <size>
           <width>391</width>
           <height>121</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="clearsend">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>清空发送</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>144</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>HEX发送</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="sendButton">
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>字符串发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
